github/resource_github_repository_test.go

```diff
@@ -841,6 +882,34 @@ resource "github_repository" "foo" {
 `, randString, randString)
 }
 
+func testAccGithubRepositoryCreateFromTemplate(randString string) string {
+	return fmt.Sprintf(`
+resource "github_repository" "foo" {
+  name         = "tf-acc-test-%s"
+  description  = "Terraform acceptance tests %s"
+  homepage_url = "http://example.com/"
+
+	template {
+		# FIXME: Change this to something more suitable for CI runs
+		owner = "jcudit"
```

Comment:
@paultyng - If there is a better repository or organization to use, please let me know.